game.js

```diff
@@ -56,6 +67,23 @@ class Game {
 	}
 
 	// Draw the game screen(s)
+  resetGame() {
```

Comment:
What do you intend `reset` to mean?  Reset everything, including the original list of phrases?

Reply:
I removed this function